tools/cli/internal/cli/merge/merge.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2024 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package merge
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/tufin/oasdiff/load"
+)
+
+type Opts struct {
+	Base       *load.SpecInfo
+	outputPath string
+}
+
+func (o *Opts) Run(args []string) error {
+	// To add in follow up PR: CLOUDP-225849
+	return nil
+}
+
+func (o *Opts) removeExternalReferences(paths []string, federated *load.SpecInfo) ([]byte, error) {
+	// To add in follow up PR: CLOUDP-225849
+	return nil, nil
+
+}
+func (o *Opts) saveFile(data []byte) error {
+	if err := os.WriteFile(o.outputPath, data, 0644); err != nil {
+		return err
+	}
+	_, _ = fmt.Printf("\nFederated Spec was saved in '%s'\n", o.outputPath)
+	return nil
+}
+
+func (o *Opts) PreRunE(args []string) error {
+	// To Add in follow up PR: CLOUDP-225849
+	return nil
+}
+
+func Builder() *cobra.Command {
+	opts := &Opts{}
+
+	cmd := &cobra.Command{
+		Use:   "merge [base-spec] [spec-1] [spec-2] [spec-3] ... [spec-n]",
```

Comment:
let;s use flags, they are friendlier

Reply:
I will add the logic in a follow-up PR. Out of curiosity, how do you envision the usage of the command when we want to merge multiple spec into the base one? Based on your comment, I was thinking about moving the base spec to a flag and leaving the other as args. What do you think?

Reply:
either same as you have here, first is base or `--base` `--external...` or something like that and why flags tend to be friendlier